content/knowledge-base/guides/composition-revisions.md

```diff
@@ -1,15 +1,13 @@
 ---
 title: Composition Revisions
-weight: 100
+state: alpha
```

Comment:
These are going to beta with v1.11. We'll need to update this before that release is cut.

Reply:
Updated with 896dda3bd91e506f8b450ec6c61a991d165d1596